src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -750,6 +766,45 @@ public function links()
     }
 
     /**
+     * Returns an Image object for the first node in the list.
+     *
+     * @param string $method The method for the image (get by default)
+     *
+     * @return Image An Image instance
+     *
+     * @throws \InvalidArgumentException If the current node list is empty
+     *
+     * @api
+     */
+    public function image($method = 'get')
```

Comment:
Probably not. Should I remove the `$method` parameter here? It's still needed for `AbstractUriElement` but I could hardcode the `get`.